### ISSUE TITLE
Accessibility, markup & CSS improvements

### ISSUE THREAD

**Issue by chrisvxd** [measuredco/puck#269]
Following #247, we should do a pass to address accessibility, markup and CSS issues, and related problems.

This is blocked by #247 and #265. It blocks #139, which should be done with CSS properties.

## Tasks

* [ ] Sort CSS properties
* [ ] Systematise hex colors with alpha channel, or comment on them
* [ ] Consistent sorting of imports

**Comment by chrisvxd** [comment]
@monospaced let's use this to track and implement broad improvements. Please drop a comment with anything you've already identified for improvement, or anything that comes up during #247.

**Comment by adampage** [comment]
For the [accessibility audit](https://github.com/measuredco/puck/issues/269#:~:text=breakpoints%C2%A0%23372-,Accessibility%20audit,-Contrast), I‘d recommend having a look at the W3C’s [Authoring Tool Accessibility Guidelines (ATAG)](https://www.w3.org/WAI/standards-guidelines/atag/) standard. ATAG is a nice companion to [WCAG](https://www.w3.org/WAI/WCAG22/quickref/) since it identifies some distinct success criteria applicable to CMS UIs.

Beyond its coverage for the accessibility of the interface itself, I also appreciate that ATAG has a whole section on **supporting authors** (encouraging them, even) to [**produce accessible content**](https://www.w3.org/TR/ATAG20/#principle_b2). 😅 Puck’s fantastic [heading analyzer plugin](https://github.com/measuredco/puck/tree/main/packages/plugin-heading-analyzer) is a heartening example of where you’ve already done work in this space.

Warning: ATAG was [last published in 2015](https://www.w3.org/TR/ATAG20/#principle_b2:~:text=W3C%20Recommendation-,24%20September%202015,-This%20version%3A), so please forgive it for potentially being out-of-touch with modern content management concepts and techniques.
